src/test/java/com/edu/UserTest.java

```diff
@@ -0,0 +1,34 @@
+package com.edu;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class UserTest {
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"123", "486", "387", "267", "954", "527"})
+    void validTrueTest(String testNumber) {
+        assertThatCode(() -> {
+            user.validUserNumber(testNumber);
+        }).doesNotThrowAnyException();
```

Comment:
```suggestion
        assertThatCode(() -> user.validUserNumber(testNumber))
                .doesNotThrowAnyException();
```
불필요한 중괄호를 제거하고 위와 같이 변경하는게 더 좋을 것 같습니다.